content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,25 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. warning::
+      If, after a short while, the panel is still empty. It is possible that your browser cookies
+      settings policy did prevent it to load.
+      Note that this policy does also change if you are connected in "Incognito" mode on your
+      browser.
+
+      To fix this issue, you can configure your browser to always allow cookies on Odoo's plugin page.
+
+      For Chrome in particular you can do so by following the guide at:
```

Comment:
Small changes to improve wording and punctuation.

`For Google Chrome, you can do so by following the guide at`